packages/root-cms/ui/pages/DocumentPage/DocumentPage.tsx

```diff
@@ -380,7 +490,11 @@ DocumentPage.Preview = (props: PreviewProps) => {
           </div>
         )}
         <div className="DocumentPage__main__previewFrame__iframeWrap">
-          <iframe ref={iframeRef} src={previewUrl} />
+          <iframe
+            ref={iframeRef}
+            src={props.allowIframePreview ? previewUrl : undefined}
```

Comment:
I dont' think the iframe tag should be output at all if the "allowIframePreview" is false